rctab_cli/sub_apps/__init__.py

```diff
@@ -1,3 +1,4 @@
+"""Subscription management app."""
 from .sub import subscription_app
```

Comment:
Did we agree on using absolute imports?

Reply:
Yep, would be good to add the pylint absolute import extension to all rctab repos.

Reply:
https://github.com/alan-turing-institute/RCTab/issues/9